iris_grib/_save_rules.py

```diff
@@ -1302,6 +1401,9 @@ def product_definition_section(cube, grib, full3d_cube=None):
         elif 'WMO_constituent_type' in cube.attributes:
             # forecast for atmospheric chemical constiuent (template 4.40)
             product_definition_template_40(cube, grib, full3d_cube)
+        elif 'spatial_processing_type' in cube.attributes:
```

Comment:
Yep.  PR #129 helps identify some issues which are not picked up by these tests because they're mocks (I hate mock so much...), so maybe a further PR following that one might add even more tests and checks, but maybe not.  We shall see.